hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -204,17 +204,6 @@
       This volume choosing policy selects volumes in a round-robin order.
     </description>
   </property>
-  <property>
```

Comment:
@Devesh, why these properties are removed?

Reply:
> @devesh, why these properties are removed?

It was removed by git while doing AUTO-MERGING. Added back and confirmed execution of all test cases passing in local